src/app/admin/admin-issuer/admin-issuer.component.html

```diff
@@ -38,23 +38,41 @@
-                        text="Download investors report"
-                        [onClick]="downloadAdminInvestorsReport.bind(this)">
-                  <ng-template #content>
-                    <svg class="w-3 h-3" width="22" height="18" viewBox="0 0 22 18" fill="none"
-                         xmlns="http://www.w3.org/2000/svg">
-                      <path d="M1.47461 13V14C1.47461 14.7956 1.85599 15.5587 2.53485 16.1213C3.21371 16.6839 4.13444
+              <div class="flex flex-col items-center lg:flex-row gap-2">
+                <div>
+                  <button app-action-button
+                          class="flex items-center justify-between gap-2 px-4 py-0.5 text-sm
+                                 bg-indigo-200 hover:bg-indigo-300 shadow-sm rounded-full transition-all"
+                          text="Download investors report"
+                          [onClick]="downloadAdminInvestorsReport.bind(this)">
+                    <ng-template #content>
+                      <svg class="w-3 h-3" width="22" height="18" viewBox="0 0 22 18" fill="none"
+                           xmlns="http://www.w3.org/2000/svg">
+                        <path d="M1.47461 13V14C1.47461 14.7956 1.85599 15.5587 2.53485 16.1213C3.21371 16.6839 4.13444
                                  17 5.0945 17H17.1608C18.1209 17 19.0416 16.6839 19.7204 16.1213C20.3993 15.5587 20.7807
                                  14.7956 20.7807 14V13M15.9542 9L11.1276 13M11.1276 13L6.30113 9M11.1276 13V1"
+                              stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                      </svg>
+                      <span>Download investors report</span>
+                    </ng-template>
+                  </button>
+                </div>
+
+                <div>
+                  <button class="flex items-center justify-between gap-2 px-4 py-0.5 text-sm
+                                 bg-indigo-200 hover:bg-indigo-300 shadow-sm rounded-full transition-all"
+                          routerLink="edit-campaign-visibility">
+                    <svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg"
+                         class="w-3 h-3">
+                      <path d="M11.6933 3.36027L14.64 6.30694M12.9433 2.11027C13.3341 1.71952 13.8641 1.5 14.4167
+                             1.5C14.9693 1.5 15.4992 1.71952 15.89 2.11027C16.2808 2.50103 16.5003 3.031 16.5003
+                             3.58361C16.5003 4.13621 16.2808 4.66619 15.89 5.05694L4.41667 16.5303H1.5V13.5536L12.9433
+                             2.11027Z"
                             stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
                     </svg>
-                    <span>Download investors report</span>
-                  </ng-template>
-                </button>
+                    <span>Manage visibility of campaigns</span>
```

Comment:
Can you rename it to "Manage offers visibility"? We have to distinct campaigns and offers. Offers are campaigns visible to the users.